src/chrome/content/rules/mathjax.xml

```diff
@@ -8,6 +8,8 @@
 
     <target host="mathjax.org" />
```

Comment:
Please also add `www.mathjax.org` and remove the outdated comment on the top.
I think we can also remove the `problem-https` link and the `(partial)` flag in the ruleset name.
Please update the code style to use only tabs to indent lines.